packages/terra-application-utility/docs/README.md

```diff
@@ -11,9 +11,24 @@
 
 ```jsx
 import React from 'react';
-import ApplicationUtility from 'terra-application-utility';
+import { ApplicationHeaderUtility, ApplicationMenuUtility } from 'terra-application-utility';
+
+<ApplicationHeaderUtility
+  menuItems={menuItems}
+  onChange={this.handleOnChange}
+  onDisclose={this.onDiscloseUtility}
+  title={userName}
+  accessory={image}
+/>
+
+<ApplicationMenuUtility
+  menuItems={menuItems}
+  onChange={this.handleOnChange}
+  onDisclose={this.onDiscloseUtility}
+  title={userName}
+  accessory={image}
+/>
 
-<ApplicationUtility {props...} />
 ```
 
 ## Component Features
```

Comment:
You should be able to uncomment all of these. Under the cross-browser support, I'd add a bullet point that [babel-polyfill](https://babeljs.io/docs/usage/polyfill) is needed to support usage of the Map object in IE.

Reply:
[Done](https://github.com/cerner/terra-framework/pull/103/commits/efd64c3eed5f7146aaefa0505339e34e92f75bed#diff-c2a019a5d2aa1bb5a4f83063a9a651e0R78).